aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -399,9 +419,14 @@ Resources:
       RecordSets:
         - Name: "<%=subdomain('origin')%>"
           Type: A
+<%   if load_balancer -%>
           AliasTarget:
             HostedZoneId: !GetAtt [LoadBalancer, CanonicalHostedZoneNameID]
             DNSName: !GetAtt [LoadBalancer, CanonicalHostedZoneName]
+<%   else -%>
+          TTL: 60
+          ResourceRecords: [!GetAtt Daemon.PublicIp]
+<%   end -%>
```

Comment:
For standalone adhoc instances without a load-balancer, the DNS entries point directly to the standalone `daemon` EC2 instance, so the added conditions are needed for this case.